terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -188,6 +188,7 @@ data "aws_iam_policy_document" "assume_role_policy" {
     principals {
       type = "AWS"
       identifiers = ["arn:aws:iam::${local.modernisation_platform_account.id}:root",
+        "arn:aws:iam::${local.environment_management.account_ids[terraform.workspace]}:role/github-actions",
```

Comment:
So this is the right code but in the wrong place...this is going to be a little more complex than we initially thought.

This code is for the MemberInfrastructureAccess role created only in the testing-test account.

We actually need to add this line in the role created in all the other member accounts which is here - 

https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/bootstrap/delegate-access/iam.tf#L33

But that is created by an external module which assigns the assume role here:
https://github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access/blob/main/main.tf#L2

And that is used in multiple places...so we would need to either pass in the policy or do some sort of conditional logic.  I seem to remember this is why I didn't use this module for the role in the testing-test account because there was no way I could allow the testing-ci user to assume it.

Reply:
OK! I've address this through this PR: https://github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access/pull/7
This allows me to supply `additional_trust_roles` through which I can pass the relevant `github-actions` role.